03-task-manager/starter/db/connect.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+
+
+const connectDB = (url) => {
+  return mongoose.connect(url, {
+    useNewUrlParser: true,
+    useCreateIndex: true,
+    useFindAndModify: false,
+    useUnifiedTopology: true,
```

Comment:
Once/if you update your Mongoose version, you will want to remove these options (as they'll no longer work and throw errors and/or warnings).